coursemapper-kg/concept-map/src/pipelines/modify_graph.py

```
@@ -17,7 +17,8 @@ def remove_concept(self, material_id: str, concept_id: str):
     def remove_concept_by_name(self, material_id: str, concept_name: str):
         self._graph_db.remove_concept_by_name(material_id, concept_name)
 
-    def add_concept(self, material_id: str, concept_name: str, slides: list[int] | None):
+    def add_concept(self, material_id: str, concept_name: str, slides: list[int]| None, isNew:bool, isEditing:bool, lastEdited:bool) :
```

Ruff (0.12.2)

Missing return type annotation for public function `add_concept`

Add return type annotation: `None`

(ANN201)